Makefile

```diff
@@ -10,22 +10,25 @@ MANPREFIX = $(PREFIX)/share/man
 AUTORELOAD = inotify
 
 # enable features requiring giflib (-lgif)
-HAVE_GIFLIB = 1
-
+HAVE_LIBGIF ?= 0
```

Comment:
If we're going to rename this, I think it's best to rename the `#ifdef`s inside the source as well for consistency.